_includes/markdown/Tools.md

```diff
@@ -22,14 +22,16 @@ is `docker.test`, you can access PHPMemcachedAdmin at `docker.test/__memcacheadm
 
 ## Xdebug
 
-Xdebug is included in the php images and is nearly ready to go out of the box. Make sure your IDE is listening for
+Xdebug is included in the php images but must be manually enabled.  To enable Xdebug, set the enviornment variable
+`ENABLE_XDEBUG` to `'true'` in the `docker-compose.yml` file in the root of the project. Make sure your IDE is listening for
```

Comment:
It will be changed in the `2.8.0` version. `XDEBUG` will be enabled for new environments created with local docker 2.8.0+. So, I think we need to rephrase this a little bit to let users know that xdebug might be not activated if they created an environment before 2.8.0 and they need to manually enable it.

Reply:
@eugene-manuilov are you good to make any final changes here and merge them in so the docs reflect v2.8.1?

Reply:
Yep, all good now.